src/Dfc.CourseDirectory.WebV2/Behaviors/Errors/NoAliasDefined.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace Dfc.CourseDirectory.WebV2.Behaviors.Errors
+{
+    public struct NoAliasDefined
```

Comment:
I'm not wild about this error mechanism but keeping in line with what's already done here.